src/cpu/x64/jit_uni_layer_normalization.cpp

```diff
@@ -1096,8 +1107,10 @@ struct jit_diff_data_base_kernel_t : diff_data_kernel_t,
         mov(reg_diff_src, ptr[reg_param + PARAM_OFF(diff_src)]);
         mov(reg_scale, ptr[reg_param + PARAM_OFF(ss)]);
 
-        if (calculate_diff_stats_)
+        // todo: investigate later
```

Comment:
It's TODO, capitalized, it is recognized this way by many IDEs.

BTW, what's this TODO about?

Reply:
oh that's just traces of figuring out the code, I'll clean up, good catch!